src/error.ts

```diff
@@ -0,0 +1,103 @@
+import { inspect } from 'node:util'
```

Comment:
Is there a way to make this not depend on Node.js? `node:util` is not available in browsers.